ai/executors/pathfinder_module.py

```diff
@@ -51,7 +51,7 @@ def get_pertinent_collision_objects(commanded_player, game_state, ai_command):
         if dist_commanded_to_other + dist_target_to_other < dist_commanded_to_target * factor:
             collision_bodies.append(CollisionBody(other.pose.position, other.velocity.position, gap_proxy))
 
-    if ai_command.ball_collision and game_state.ball:
+    if ai_command.ball_collision and game_state.ball_on_field:
```

Comment:
Nitpickess of nitpick: `is_ball_on_field`

Reply:
oups my bad. Je vais fix.